.github/CODEOWNERS

```diff
@@ -0,0 +1,60 @@
+#	This file sets up default notification for reviews based on who "owns" code.
+
+/run					@weiwangncar
+/configure				@davegill @llpcarson @liujake
+/compile				@davegill @llpcarson @liujake
+/clean					@davegill @llpcarson @liujake
+Makefile				@davegill
+
+#	Architecture directories
+
+/arch					@davegill
+/tools					@davegill
+
+#	Chemistry
+/chem					@ravanah @kayeekayee @stacywalters
+
+#	Physics
+
+/phys					@dudhia @weiwangncar @llpcarson
+/phys/module_mp_thompson.F		@gthompsnWRF
+
+#	Data Assimilation
+
+/wrftladj				@liujake
```

Comment:
@jjguerrette 
JJ,
Should you be also a code owner for the wrftladj and var directories?

Reply:
I think @liujake would like that in the future, but I am not sure if we are ready for that transition.  Let's ask when he is back in town.